plugin/styleguide-plugin.js

```diff
@@ -66,7 +66,7 @@ StyleguidePlugin.prototype.apply = function apply(compiler) {
 
   compiler.plugin('emit', (compilation, callback) => {
     // Get the bundled Styleguide Client
-    const clientApi = fs.readFileSync(path.join(__dirname, 'client-api.js'));
+    const clientApi = fs.readFileSync(path.join(__dirname, '../client', 'client-api.js'));
```

Comment:
This won't work as client-api is es6 now.
But its a good start point - I'll try to finish this tonight